CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Version 1.13.1 (unreleased)
+
+### 🛠️ Bug fixes
+
+- Fix duplicate class error on Android build
+  ([#5882](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/5882))
+
```

Comment:
```suggestion
- Fix spring-kafka single record instrumentation 
  ([#5904](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/5904))

```

Reply:
oh, thx! I didn't fetch upstream before running this locally:

```
git format-patch --stdout HEAD..upstream/$GITHUB_REF_NAME CHANGELOG.md | git apply --3way
```